tcms/testruns/migrations/0006_rename_test_case_run_to_test_execution.py

```diff
@@ -5,9 +5,9 @@
 
 
 def rename_permissions(apps, schema_editor):
-    Permission = apps.get_model('auth', 'Permission')
+    permission = apps.get_model('auth', 'Permission')
 
-    for permission in Permission.objects.filter(codename__contains='testcaserun'):
+    for permission in permission.objects.filter(codename__contains='testcaserun'):
```

Comment:
Same comment as above about loop variable name vs. model name. Use "permission_model" or something like that.